website/questmaker/quest.py

```diff
@@ -406,6 +421,16 @@ def to_dict(self):
                 'pos_x': self.pos_x,
                 'pos_y': self.pos_y}
 
+    def to_db(self):
```

Comment:
This function doesn't write question to db. Rename it or refactor, e.g.  add set_question inside this function.